Watchman.DomainModel/Settings/ConfigurationItems/MinAverageMessagesPerWeek.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Watchman.DomainModel.Settings.ConfigurationItems
+{
+    public class MinAverageMessagesPerWeek : MappedConfiguration<int>
+    {
+        public override string Name { get; } = nameof(MinAverageMessagesPerWeek);
```

Comment:
myśle czy to nameof da sie jakoś zautomatyzować
może w konstruktorze refleksją w klasie z której dziedziczymy

ale wersje taką jak tutaj widziałem dużo razy
myśle że jest ok